tests/behat/features/bootstrap/RaContext.php

```diff
@@ -290,6 +291,62 @@ public function iShouldSeeInAuditLogIdentityOverview($institution)
         }
     }
 
+    /**
+     * @Given I remove token from user arg1
+     */
+    public function removeTokenFromUser($name)
+    {
+        $this->removeTokenWithIdentifierFromUser('03945859', $name);
+    }
+
+    /**
+     * @Then I remove token with identifier :arg2 from user :arg1
+     */
+    public function removeTokenWithIdentifierFromUser($identifier, $name)
+    {
+        $page = $this->minkContext->getSession()->getPage();
+
+        // Get row
+        /** @var NodeElement[] $searchResults */
+        $searchResults = $page->findAll('xpath', sprintf("//td[contains(.,'%s')]/..", $name));
+        if (empty($searchResults)) {
+            throw new Exception(
+                sprintf('No tokens found for user "%s"', $name)
+            );
+        }
+        $token = null;
+        foreach ($searchResults as $searchResult) {
+            $button = $searchResult->find('css', '.btn.revoke');
+            if ($button) {
+                $attribute = $button->getAttribute("data-sfidentifier");
+                if ($attribute === $identifier) {
+                    $token = $attribute;
+                    break;
+                }
+            }
+        }
```

Comment:
Why not use find here?         $searchResult = $page->find('css', sprintf('[data-sfidentifier=%s]', json_encode($identifier)));